pkg/github/runners/manager.go

```diff
@@ -140,7 +140,7 @@ func (m *RunnerManager) ProcessMessages(ctx context.Context, handler func(msg *t
 	for {
 		message, err := m.messageQueueManager.ReceiveNextMessage(ctx, m.lastMessageId)
 		if err != nil {
-			return fmt.Errorf("unable to get the next message from the message queue. %w", err)
+			m.logger.Errorf("unable to get the next message from the message queue. %w", err)
```

Comment:
I believe if the context is cancelled we should return the error, so we can exit.
If the context is cancelled this means someone is stopping the application.
Currently with this change we are going to log `Unable to get the next message from the message queue` until the app closes gracefully.

So we could add to avoid it:
```
  if ctx.Err() == context.Canceled {
  	return err
  }
```